src/ui/components/ObjectView.vue

```diff
@@ -222,7 +222,12 @@ export default {
                 this.actionCollection.destroy();
             }
 
-            this.actionCollection = this.openmct.actions._get(this.currentObjectPath || this.objectPath, this.currentView);
+            const options = {
```

Comment:
tried to add something like this,
```
const wrappedView = provider.view;
        provider.view = function(domainObject, objectPath) {
            const viewObject = wrappedView(domainObject, objectPath);
            const wrappedShow = viewObject.show;
            viewObject.show = function(element) {
                viewObject.element = element;
                wrappedShow(element);
            }

            return viewObject;
        };
```

but in line `this.actionCollection=` we are passing view (this.currentView) and due to race condition 'show' function is not called yet and thats why view.element is `undefined` and action `appliesTo` function cant use `view.element` properly :(

Reply:
In that case, `getActionCollection` is being called too early. Does it work it if we move it below the call to `.show(...)`?

Reply:
With these changes, in case of objectView `show(element)` the element has class 'l-angular-ov-wrapper',
this works for old views (eg: plots), but I think in new views it replaces this element with component.
so I get element that is not in DOM.

Solved by using parentelement instead.
`viewObject.parentElement = element.parentElement;`

Plots:
<img width="444" alt="Screen Shot 2021-05-13 at 12 42 13 PM" src="https://user-images.githubusercontent.com/1292612/118178483-d9099300-b3e8-11eb-878d-42ea9d152939.png">

Imagery:
<img width="544" alt="Screen Shot 2021-05-13 at 12 42 33 PM" src="https://user-images.githubusercontent.com/1292612/118178486-d9a22980-b3e8-11eb-942f-7f6c7d463582.png">